lib/solaar/cli/__init__.py

```diff
@@ -98,7 +98,7 @@ def _find_receiver(receivers, name):
 	assert name
 
 	for r in receivers:
-		if name in r.name.lower() or name == r.serial.lower():
+		if name in r.name.lower() or ( (name == r.serial.lower()) if (r.serial is not None) else False) :
```

Comment:
```python
if name in r.name.lower() or (r.serial and name == r.serial.lower()):
```

This should work but please test.

Reply:
I'm going to use is not None to better handle the edge case of an empty string for a serial number.

Reply:
OK, I amended and force-pushed the change so that there is only one commit.

Reply:
An empty string will also evaluate as false. I'd prefer `r.serial` instead of `r.serial is None` as that is simpler but I'll accept either way.

Reply:
I actually used r.serial is None so that falsy serial numbers can be checked.  I don't expect it to ever happen, but I didn't want the code to be tripped up in this vanishingly unlikely situation.